src/pages/components/localemodal.mdx

```diff
@@ -28,12 +28,14 @@ The Locale modal provides the option to select a geographic regions and language
 
 ## Design and functional specifications
 
-The design specs and functional specs for the Locale modal can be viewed <a href="https://ibm.box.com/s/6eyktf1gti4kfg6s06ozjx43oa0h6k06" target="_blank">here</a>.
+The design specs and functional specs for the Locale modal can be viewed <a href="https://ibm.box.com/s/nwy0fyiw6mu3qwd3p882pnqmit93rfa5" target="_blank">here</a>.
 
 ## Development documentation
 
 <ComponentStatus name="Locale Modal" type="ui" />
 
+https://ibm.box.com/s/nwy0fyiw6mu3qwd3p882pnqmit93rfa5
```

Comment:
@JennySanchez did you mean to add the link here?

Reply:
@jeffchew no...